test/e2e/traceflow_test.go

```diff
@@ -1102,27 +1138,42 @@ func testTraceflowInterNode(t *testing.T, data *TestData) {
 
 	// Create Service backend Pod. The "hairpin" testcases require the Service to have a single backend Pod,
 	// and no more, in order to be deterministic.
-	nginxPodName := "nginx"
-	require.NoError(t, data.createNginxPodOnNode(nginxPodName, testNamespace, node2, false))
-	nginxIP, err := data.podWaitForIPs(defaultTimeout, nginxPodName, testNamespace)
+	agnhostPodName := "agnhost"
+	mutateFunc := func(pod *corev1.Pod) {
+		pod.Labels["app"] = "agnhost-server"
+	}
+	require.NoError(t, data.createPodOnNode(agnhostPodName, testNamespace, node2, agnhostImage, []string{"sleep", strconv.Itoa(3600)}, nil, nil, nil, false, mutateFunc))
+	agnhostIP, err := data.podWaitForIPs(defaultTimeout, agnhostPodName, testNamespace)
 	require.NoError(t, err)
 
-	var nginxIPv4Str, nginxIPv6Str, svcIPv4Name, svcIPv6Name string
-	if nginxIP.ipv4 != nil {
-		nginxIPv4Str = nginxIP.ipv4.String()
+	var agnhostIPv4Str, agnhostIPv6Str, svcIPv4Name, svcIPv6Name string
+	if agnhostIP.ipv4 != nil {
+		agnhostIPv4Str = agnhostIP.ipv4.String()
 		ipv4Protocol := corev1.IPv4Protocol
-		svcIPv4, err := data.createNginxClusterIPService("nginx-ipv4", testNamespace, false, &ipv4Protocol)
+		svcIPv4, err := data.CreateService("agnhost-ipv4", testNamespace, 80, 8080, map[string]string{"app": "agnhost-server"}, false, false, corev1.ServiceTypeClusterIP, &ipv4Protocol)
 		require.NoError(t, err)
 		svcIPv4Name = svcIPv4.Name
 	}
-	if nginxIP.ipv6 != nil {
-		nginxIPv6Str = nginxIP.ipv6.String()
+	if agnhostIP.ipv6 != nil {
+		agnhostIPv6Str = agnhostIP.ipv6.String()
 		ipv6Protocol := corev1.IPv6Protocol
-		svcIPv6, err := data.createNginxClusterIPService("nginx-ipv6", testNamespace, false, &ipv6Protocol)
+		svcIPv6, err := data.CreateService("agnhost-ipv6", testNamespace, 80, 8080, map[string]string{"app": "agnhost-server"}, false, false, corev1.ServiceTypeClusterIP, &ipv6Protocol)
 		require.NoError(t, err)
 		svcIPv6Name = svcIPv6.Name
 	}
 
+	// Mesh ping to activate tunnel on Windows Node
```

Comment:
Please add todo here to explain this is a workaround, and will be removed after Windows OVS fixes the issue that first packet is possibly dropped on tunnel because the ARP entry doesn't exist in host cache. The OVS issue link is: https://github.com/openvswitch/ovs-issues/issues/253

Reply:
Thanks. Added TODO here.